pubtools/pulplib/_impl/client/client.py

```diff
@@ -203,6 +203,36 @@ def search_repository(self, criteria=None):
             Repository, "repositories", criteria=criteria, search_options=search_options
         )
 
+    def search_repository_content(self, repository_id, type_ids, criteria=None):
+        """Search the given repository for content matching the given criteria.
+
+        Args:
+            repository_id (str)
+                The ID of the repository in which to search
```

Comment:
minor: may be something like "repository ID to search"

Reply:
Well, that would make sense but we're searching for content **within** the repo with the given ID.

Reply:
either is fine as long as it's understood. just thought too many linkers in the statement.